gateway.tf

```diff
@@ -166,13 +298,25 @@ resource "aws_api_gateway_method_response" "api_proxy_any" {
   resource_id = aws_api_gateway_resource.api_proxy.id
   http_method = aws_api_gateway_method.api_proxy_any.http_method
   status_code = "200"
+
+  response_parameters = {
+    "method.response.header.access-control-allow-headers" = true,
+    "method.response.header.access-control-allow-methods" = true,
+    "method.response.header.access-control-allow-origin" = true
+  }
 }
 
 resource "aws_api_gateway_integration_response" "api_proxy_any_integration" {
   rest_api_id = aws_api_gateway_rest_api.main.id
   resource_id = aws_api_gateway_resource.api_proxy.id
   http_method = aws_api_gateway_method.api_proxy_any.http_method
   status_code = aws_api_gateway_method_response.api_proxy_any.status_code
+  
+  response_parameters = {
+    "method.response.header.access-control-allow-headers" = "'Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token'",
+    "method.response.header.access-control-allow-methods" = "'DELETE,GET,HEAD,OPTIONS,PATCH,POST,PUT'",
+    "method.response.header.access-control-allow-origin" = "'*'"
```

Comment:
@leorossi we need to map the header, not pass back `*` can you look into this?

Reply:
I removed this block entirely, it seems meaningless because the integration is an HTTP_PROXY (in fact also in the console UI you can't set those headers).